.circleci/config.yml

```diff
@@ -539,6 +540,7 @@ jobs:
             source << parameters.sg_new_env_name >>/bin/activate
             python3.8 -m pip install --upgrade setuptools pip wheel
             python3.8 -m pip install -r requirements.txt
+            python3.8 -m pip install -r requirements.dev.txt
```

Comment:
We have a few test checking the scores of our mAP implementation vs COCOEval. For this reason we need pycocotools installed when running tests. So that's why it is added to dev dependencies